bamboo-specs/bamboo.yml

```diff
@@ -27,7 +27,7 @@ Build:
     - script: |
         #!/usr/env/bin bash
         set -e
-        docker-compose -f test/docker-compose.yml up --exit-code-from tests
+        docker-compose -f test/docker-compose.yml up --scale selenium-chrome=3 --scale selenium-firefox=3 --exit-code-from tests
```

Comment:
Waarom dit? Start hij testen in parallel op?

Reply:
We ondervonden flakyness bij het afsluiten van Chrome en/of Firefox in Docker, vermoedelijk door het draaien van unit testen en e2e testen dat die elkaars browsers afsluiten tijdens de testen. Door het openen van enkele instanties, blijkt dit niet meer voor te komen. Dit is een tijdelijke oplossing en wordt later verder onderzocht.